modules/virt-maintain-high-availability.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-maintain-high-availability_{context}"]
+= Maintaining high availability of virtual machines
+
+There are three options to maintain high availability (HA) of virtual machines:
```

Comment:
What about restructuring the bullets to give easier guidance:
1. Automatic high availability for IPI is provided via MHC
2. Automatic high availability for non-IPI (and also alternatively for IPI) is provided for NHC
3. Alternatively high availability for any platform can be provided through the use either a monitoring system or a qualified human to monitor node availability

Reply:
@fabiand Take a look at the updates.

Reply:
Looks great to me, thanks!